src/fuzzer.py

```diff
@@ -136,10 +136,10 @@ def fuzzing_round(self, test_case: TestCase, inputs: List[Input]) -> Optional[Eq
         # at this point we need to increase the effectiveness of inputs
         # so that we can detect contract violations (note that it wasn't necessary
         # up to this point because we weren't testing against a contract)
-        boosted_inputs: List[Input] = self.boost_inputs(inputs, 1)
+        boosted_inputs: List[Input]
 
         # check for violations
-        ctraces = self.model.trace_test_case(boosted_inputs, 1)
+        ctraces, boosted_inputs = self.trace_and_boost(inputs, 1)        
```

Comment:
Could you move this call up by two lines? Otherwise, the comment at lines 136-138 makes no sense.

Reply:
done